commands/databases_test.go

```diff
@@ -858,3 +874,58 @@ func TestDatabasesReplicaDelete(t *testing.T) {
 		assert.EqualError(t, err, errTest.Error())
 	})
 }
+
+func TestDatabaseGetSQLModes(t *testing.T) {
+	// Successful
+	t.Run("Success", func(t *testing.T) {
+		withTestClient(t, func(config *CmdConfig, tm *tcMocks) {
+			tm.databases.EXPECT().GetSQLMode(testDBCluster.ID).Return(testSQLModes, nil)
+
+			config.Args = append(config.Args, testDBCluster.ID)
+
+			err := RunDatabaseGetSQLModes(config)
+			assert.NoError(t, err)
+		})
+	})
+
+	// Error
+	t.Run("Error", func(t *testing.T) {
+		withTestClient(t, func(config *CmdConfig, tm *tcMocks) {
+			tm.databases.EXPECT().GetSQLMode(testDBCluster.ID).Return(nil, errTest)
+
+			config.Args = append(config.Args, testDBCluster.ID)
+
+			err := RunDatabaseGetSQLModes(config)
+			assert.Error(t, err)
+		})
+	})
+}
+
+func TestDatabaseSetSQLModes(t *testing.T) {
+	testSQLModesInterface := []interface{}{}
+	for _, sqlMode := range testSQLModes {
+		testSQLModesInterface = append(testSQLModesInterface, sqlMode)
+	}
+
+	// Successful
```

Comment:
lets add the `t.Runs` here as well and remove the comments.

Reply:
*scratches head* I remember doing that one too 0.o